resetkrun.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+(shopt -s nullglob; rm -rf luajit*{log,manifest,bz2,envlogs,env,instr_data})
+(shopt -s nullglob; rm -rf quicktest*{log,manifest,bz2,envlogs,env,instr_data})
```

Comment:
Ah, `shopt -s nullglob`. This seems to be required since the update to Debian 9. Annoying.